http-client/Network/HTTP/Client/Request.hs

```diff
@@ -350,6 +352,13 @@ addProxy :: S.ByteString -> Int -> Request -> Request
 addProxy hst prt req =
     req { proxy = Just $ Proxy hst prt }
 
+
+-- | Send secure requests to the proxy in plain text rather than using CONNECT.
+--
+-- @since 0.7.2
+addProxyPlainTextNoCONNECTMode :: Request -> Request
```

Comment:
I think this naming should match `useProxySecureWithoutConnect `

Reply:
Sorry, I missed that one from my git add somehow... I've pushed the changes I already had locally now, and also changed Headers.hs to re-add `Data.Monoid` (I was hoping for a clean build with `-Wall -Werror` but it seems that will break compatibility).